src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -111,72 +88,14 @@ public void run() {
-                                    final GhThread thread = user.thread(upd.getCallbackQuery().getData().split("#")[1]);
-                                    final Issue issue = new ThreadIssue(user.github(), thread);
-                                    return new RsText(
-                                        new FormattedText(
-                                            "[#%d](%s) - %s\n\n%s",
-                                            issue.number(),
-                                            String.format("https://github.com/%s/issues/%d", issue.repo().coordinates(), issue.number()),
-                                            new Issue.Smart(issue).title(),
-                                            new Joined(
-                                                new TextOf("\n\n"),
-                                                new Mapped<>(
-                                                    cmt -> new FormattedText(
-                                                        "[@%s](https://github.com/%s) > %s",
-                                                        cmt.author().login(),
-                                                        cmt.author().login(),
-                                                        cmt.body()
-                                                    ),
-                                                    new Mapped<>(
-                                                        Comment.Smart::new,
-                                                        issue.comments().iterate(new Date(thread.lastRead().toEpochMilli()))
-                                                    )
-                                                )
-                                            )
-                                        )
-                                    );
-                                }
+                                    new TkNotificationsFallback(this.data)
                             )
                         ),
-                        (upd, err) -> {
-                            final Optional<TmResponse> rsp;
-                            if (err instanceof GhAuthException) {
-                                rsp = Optional.of(
-                                    new RsText(
-                                        new FormattedText(
-                                            "You need to [login with Github](http://%s/auth?ps=%d",
-                                            System.getenv("APP_HOST"),
-                                            GhAuthException.class.cast(err).user().uid()
-                                        )
-                                    )
-                                );
-                            } else {
-                                rsp = Optional.empty();
-                            }
-                            return rsp;
-                        }
+                        new FbFallback()
```

Comment:
@samihibrahim same here: this class is a fallback for unauthorized access: `FbFallback` → `FbUnauthorized`

Reply:
@g4s8 Fixed.